Notify/Notify/Notify.Android/Notifications/NotificationHelper.cs

```diff
@@ -60,6 +61,7 @@ public Notification GetServiceArrivedToDestinationNotification()
                 .SetContentTitle("Ofir testing")
```

Comment:
Do you know what the title is used for?
Also, I remember there was `.SetIcon()` or something like that, maybe its related to the "cross" symbol I sent you?

Reply:
It was just for testing. As far as I know of, it doesn't do anything special. Can be removed.